src/main/java/org/codinjutsu/tools/jenkins/view/extension/JobParameterRenderers.java

```diff
@@ -57,21 +57,27 @@ public static JobParameterComponent<String> createPasswordField(JobParameter job
     }
 
     @NotNull
-    public static JobParameterComponent<String> createTextArea(JobParameter jobParameter, String defaultValue) {
+    public static JobParameterComponent<String> createTextArea(JobParameter jobParameter, String defaultValue, boolean isReadonly = false) {
```

Comment:
This will not work on Java with default method. please add a second method for readOnly